boards/stm32l476g-disco/Makefile.include

```diff
@@ -3,11 +3,16 @@ INCLUDES += -I$(RIOTBOARD)/common/stm32/include
 
 # this board uses openocd with st-link
 PROGRAMMER ?= openocd
+
 OPENOCD_DEBUG_ADAPTER ?= stlink
 
 # This board can become un-flashable after a hardfault,
 # use connect_assert_srst to always be able to flash or reset the board.
 OPENOCD_RESET_USE_CONNECT_ASSERT_SRST ?= 1
 
-# openocd programmer is supported
-PROGRAMMERS_SUPPORTED += openocd
+# openocd and cpy2remed programmer is supported
+PROGRAMMERS_SUPPORTED += openocd cpy2remed
+
+#needed by cpy2remed
+FLASHFILE += $(HEXFILE)
```

Comment:
`makefiles/tools/cpy2remed.inc.mk` already takes care of that

Reply:
Fixed.